src/main/app/case/case.ts

```diff
@@ -200,6 +200,8 @@ export interface Case {
   //applicant1LanguagePreference?: LanguagePreference;
 
   safetyConcerns?: string;
+  childSafetyConcerns?: YesOrNo;
```

Comment:
@ShakiraKP We don't need to define fields unless we need to use them, fields are added userCase in session by default.
Can we please remove them if you are not using in sequence or any where else.

Reply:
Okay no problem, in adoption all values needed to be added to case data and added to cos api, I realise this is different here and I will no longer add these values to case data.